src/services/completions.ts

```diff
@@ -235,6 +235,7 @@ namespace ts.Completions {
                 kind: ScriptElementKind.classElement,
                 kindModifiers: undefined,
                 sortText: SortText.LocationPriority,
+                isDeprecated: undefined

```

Comment:
I think this shouldnt be new flag.. It should be part of `kindModifiers: ScriptElementKindModifier`

Reply:
And that goes everywhere.. It shouldnt be separate flag but we should use existing kindModifiers or add one wherever we need instead of adding new flag